Example/TableManager Example/TableManager Example/SectionIndexViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SectionIndexViewController.swift
+//  TableManager Example
+//
+//  Created by Uriel Battanoli on 1/10/17.
+//  Copyright © 2017 Morbix. All rights reserved.
+//
+
+import UIKit
+import TableManager
+
+class SectionIndexViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J"]
+        
+        alphabet.forEach { letter in
+            let newSection = tableView.addSection().setIndexTitle(letter)
+            
+            Fake.basicData().forEach { element in
+                let newRow = newSection.addRow()
+                
+                newRow.setConfiguration({ (row, cell, indexPath) in
```

Comment:
- Prefer use method chaining;
- Prefer use trailing closure;
- Remove extra parenthesis `row, cell, indexPath in`
- Remove extra parameters `_, cell, _ in`